GitVersion.yml

```diff
@@ -0,0 +1,28 @@
+mode: ContinuousDelivery
+increment: None
+branches:
+  master:
+    regex: master
+    tag: ''
+    increment: Patch
+    prevent-increment-of-merged-branch-version: true
+    track-merge-target: true
+    tracks-release-branches: false
+    is-release-branch: true
+    is-mainline: true
+  develop:
+    regex: develop
+    increment: None
+    tag: 'dev'
+    track-merge-target: true
+    tracks-release-branches: true
+    is-release-branch: false
+  .*:
```

Comment:
Is increment an optional config? Briefly looking at the docs it seems this should be "increment: Inherit".